modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -438,21 +438,24 @@ object EnrichmentManager {
     event: EnrichedEvent,
     iabEnrichment: Option[IabEnrichment]
   ): Either[NonEmptyList[FailureDetails.EnrichmentFailure], Option[SelfDescribingData[Json]]] =
-    iabEnrichment match {
-      case Some(iab) =>
-        event.user_ipaddress match {
-          case IPv4Regex(ipv4) if !List(null, "", s"\0").contains(event.useragent) =>
-            iab
-              .getIabContext(
-                Option(event.useragent),
-                Option(ipv4),
-                Option(event.derived_tstamp).map(EventEnrichments.fromTimestamp)
-              )
-              .map(_.some)
-          case _ => None.asRight
-        }
-      case None => None.asRight
-    }
+    if (List(null, "", s"\0").contains(event.useragent)) None.asRight
+    else
+      iabEnrichment match {
+        case Some(iab) =>
+          val ipAddressWithoutPort =  event.user_ipaddress match {
```

Comment:
```suggestion
          val ipAddressWithoutPort = event.user_ipaddress match {
```

Did it pass through scalafmt?